data/cinic10/__init__.py

```diff
@@ -0,0 +1,128 @@
+"""CINIC-10 data set.
+
+See https://datashare.is.ed.ac.uk/handle/10283/3192
+"""
+import os
+import tensorflow as tf
+from template import BaseDataSampler
+from .generate_cifar10_tfrecords import main as download
+
+HEIGHT = 32
+WIDTH = 32
+DEPTH = 3
+datashape = [HEIGHT,WIDTH,DEPTH]
+numexamples = 270000
+cinic_mean = [0.47889522, 0.47227842, 0.43047404]
+cinic_std = [0.24205776, 0.23828046, 0.25874835]
+
+
+class Cifar10DataSet(object):
```

Comment:
The class name needs to be changes.
```suggestion
class Cinic10DataSet(object):
```